tools/serverpod_cli/lib/src/analyzer/models/validation/restrictions/default.dart

```diff
@@ -120,4 +122,33 @@ class DefaultValueRestriction extends ValueRestriction {
     }
     return errors;
   }
+
+  List<SourceSpanSeverityException> _doubleValidation(
```

Comment:
Question: What happens if an int is provided? Is that OK?
I don't think that was covered in the tests, could you add it?

Reply:
Yes!